keps/README.md

```diff
@@ -41,7 +41,7 @@ See [KEP-1](0001-kubernetes-enhancement-proposal-process.md) for more details.
 
 If the KEP is mainly restricted to one SIG's purview then it should be in a KEP directory for that SIG.
 If the KEP is widely impacting much of Kubernetes, it should be put at the root of this directory.
-If in doubt ask [SIG-Architecture](../sig-architecture/README.md) and they can advise.
+If in doubt ask [SIG-Architecture](https://github.com/kubernetes/community/tree/master/sig-architecture) and they can advise.
```

Comment:
We generally prefer using the git.k8s.io links. :)

So this would become https://git.k8s.io/community/sig-architecture.

Reply:
Thanks, I didn't know. This is fixed now, please take another look.